src/analytics/Events.tsx

```diff
@@ -96,8 +96,8 @@ export enum KeylessBackupEvents {
   wallet_security_primer_get_started = 'wallet_security_primer_get_started',
   cab_setup_recovery_phrase = 'cab_setup_recovery_phrase',
   cab_sign_in_another_way = 'cab_sign_in_another_way',
-  cab_sign_in_with_google = 'cab_sign_in_with_google',
-  cab_sign_in_with_google_success = 'cab_sign_in_with_google_success',
+  cab_sign_in_with_email_start = 'cab_sign_in_with_email_start',
+  cab_sign_in_with_email_success = 'cab_sign_in_with_email_success',
```

Comment:
maybe drop the `with_email` part? in case we support other SSO providers in the future